components/LevitatingLink.js

```diff
@@ -1,6 +1,11 @@
+import smoothScroll from './SmoothScoll.js'
```

Comment:
We are already using [react-scroll](https://github.com/fisshy/react-scroll). Can it be used instead?